content/en/docs/user-documentation/layout-builder/advanced-options/_index.md

```diff
@@ -6,13 +6,40 @@ weight: -5
 
 A huge amount of configuration is available with Layout Builder components using the contributed [Layout Builder Blocks](https://www.drupal.org/project/layout_builder_blocks) module, which is included with the distribution. In addition to those configuration options, we provide an extra layer of "Y Styles" that help site builders customize their sites in an accessible and brand-compliant manner.
 
+{{< youtube S9QninI7ydM >}}
+
 ## Y Styles
 
-These options provide customizations of Layout Builder-enabled pages at the Content Type, Page, and Component level.
+These options provide customizations of Layout Builder-enabled pages at the Content Type, Page, and Component(/Block) level.
+
+Styles inherit from content types, to pages, to components. Some styles can also be overridden at each level - block styles can override page styles, which can override content type styles.
+
+```mermaid
```

Comment:
cool

Reply:
🧜‍♀️ there's support for a ton more too: https://www.docsy.dev/docs/adding-content/diagrams-and-formulae/